Makefile

```diff
@@ -1733,6 +1736,9 @@ jl/crc32c_ppc.o: util/crc32c_ppc.c
 
 jl/crc32c_ppc_asm.o: util/crc32c_ppc_asm.S
 	$(AM_V_CC)$(CC) $(CFLAGS) -c $< -o $@
+
+jl/crc32c_ppc_fast_zero_asm.o: util/crc32c_ppc_fast_zero_asm.S
```

Comment:
can you also make sure the build target `rocksdbjavastatic` works? It builds everything under "jls/" not "jl"

Reply:
@ajkr I have verified that the target rocksdbjavastatic builds everything under jls and not jl.

Reply:
I agree with that but still don't know whether the `rocksdbjavastatic` build target will work after this change. Have you tested it? My concern is we may need to add this target: "jls/util/crc32c_ppc_fast_zero_asm.o".